### PR TITLE
testing CI again

### PR DESCRIPTION
**Pull request by buffalojoec**

### CODE REVIEW

File: .github/workflows/pull-requests.yml [hunk 1]
```diff
@@ -69,6 +69,12 @@ jobs:
       - name: Build & Test
         run: pnpm build --concurrency=100%
 
+      - name: Download Test Validator Log
+        uses: actions/download-artifact@v2
+        with:
+          name: validator-log
+          path: test-ledger/validator.log
+
```
Comment (steveluscher) [inline .github/workflows/pull-requests.yml:77]: 👀

Comment (steveluscher) [reply .github/workflows/pull-requests.yml:77]: Haha. Harsh.
image

Comment (buffalojoec) [reply .github/workflows/pull-requests.yml:77]: Lmao I just saw. I'll have to update it to not skip on fail. Should have known. 🫠

**Comment by steveluscher** [comment]
I'll just leave this here. https://github.com/namespacelabs/breakpoint

**Pull request closed without merging**